### PR TITLE
[15.0][IMP] sale_order_product_recommendation: add widget units_included_widget

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT43880

ping @sergio-teruel

### CODE REVIEW

File: sale_order_product_recommendation/static/src/js/units_included_widget.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+            _onChangeInput: function (ev) {
+                this._updateValue(ev.target.value);
+            },
+
+            _onInputFocus: function (ev) {
+                ev.target.select();
+            },
+
+            _setUpdatedValue: function (value) {
```
Comment (sergio-teruel) [inline sale_order_product_recommendation/static/src/js/units_included_widget.js]: Please!! use the method _setValue instead

Comment (pilarvargas-tecnativa) [reply sale_order_product_recommendation/static/src/js/units_included_widget.js]: done! i was studying the _setValue function to use it :) thanks!

**Comment by pilarvargas-tecnativa** [comment]
@chienandalu please review :)

**Review by chienandalu — APPROVED** [review]
LGTM :+1: :)

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**